plugins/ILibPlugin/index.js

```diff
@@ -234,7 +237,9 @@ class ILibPlugin {
 			const definedConstants = {
 				ILIB_BASE_PATH: ilib.resolved,
 				ILIB_RESOURCES_PATH: resources.resolved,
-				ILIB_CACHE_ID: '__webpack_require__.ilib_cache_id'
+				ILIB_CACHE_ID: '__webpack_require__.ilib_cache_id',
+				// when `emit` is false and `ilib` is not absolute, can delare no assets
```

Comment:
I `delare` you to spell correctly in comments ;)

Reply:
That's the 🇨🇦 spelling!